Readme.md

```diff
@@ -6,17 +6,17 @@ This plugin is used to obtain information about a Heroku Postgres instance,
 that may be useful when analyzing performance issues. This includes information
 about locks, index usage, buffer cache hit ratios and vacuum statistics.
 
-### Installation
+## Installation
 
 ```bash
 $ heroku plugins:install heroku-pg-extras
 ```
 
-### Usage
+## Usage
```

Comment:
This puts these at the same header level as "Publishing" and the beta note at the bottom (H2), and makes them one level below the title above this, and allows the commands to all use H3, which is what most of them already are